app/themes/user-admin/e2e/user-list/user-list.e2e-spec.ts

```diff
@@ -49,6 +39,8 @@ describe('User List Page', function() {
 
     });
 
+    //For the pagination to work, the e2e test should have more than 10 users in the user-list
+    //If there are less than 10 users in the list, Please refer to user-admin/scripts/sql/create-e2e-user.sql
     it('should check for pagination', () => {
```

Comment:
unfortunately, i see these errors: 

```
  User List Page
    ✓ should login successfully with right email and password
[22:18:40] W/element - more than one element found for locator By(css selector, nav ul > li:first-child > a) - the first result will be used
    ✓ should lead to User List page at click of the User List link
    ✓ should show the user list table header
    ✗ should check for pagination
      - Expected '' to match 'active'.
      - Expected '' to match 'active'.
[22:18:51] W/element - more than one element found for locator By(css selector, tbody th:first-child) - the first result will be used
[22:18:54] W/element - more than one element found for locator By(css selector, tbody th:first-child) - the first result will be used
[22:18:56] W/element - more than one element found for locator By(css selector, tbody th:first-child) - the first result will be used
[22:18:56] W/element - more than one element found for locator By(css selector, tbody th:first-child) - the first result will be used
    ✓ should check for sort function
```

I also have  > 10 users in my DB, I think this is just a test failure trickling down